### DISCUSSION TITLE
Support /back for "Elevators - By Keehl254".

### DISCUSSION THREAD

**Discussion by alechretien16** [Zrips/CMI#7181] [Enhancement]
Hello, could you add support for the /back command regarding the elevators of the following plugin?
Using /back takes me to wherever the elevator took me.
https://www.spigotmc.org/resources/elevators.34229/

**Comment by mrfloris** [comment]
Yeah it's a bit strange how it works - also because /back has a 'range' setting. (don't /back to previous if previous is like within 25 blocks, or whatever).
